mailbox/api/src/main/java/org/apache/james/mailbox/MailboxSession.java

```diff
@@ -23,12 +23,57 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
+import java.util.Objects;
+
+import com.google.common.base.MoreObjects;
 
 /**
  * Mailbox session.
  */
 public interface MailboxSession {
 
+    class SessionId {
+
+        public static SessionId zero() {
```

Comment:
Why do we need 0 ?

Reply:
Maybe it's "no session id" than, it can be handled by  `Optional<SessionId>` where relevant